### PR TITLE
Release 16.0.0

### PR DESCRIPTION
**Pull request by rlepinski**
Waiting on iOS SDK 17.0.2 and then to bump the proxy version

### CODE REVIEW

File: android/gradle.properties [hunk 1]
```diff
@@ -4,8 +4,8 @@ Airship_targetSdkVersion=31
 Airship_compileSdkVersion=31
 Airship_ndkversion=21.4.7075529
 
-Airship_airshipProxyVersion=2.1.1
+Airship_airshipProxyVersion=3.0.0
```
Comment (rlepinski) [inline android/gradle.properties:7]: needs to be bumped to 3.0.1

File: android/gradle.properties [hunk 2]
```diff
@@ -4,8 +4,8 @@ Airship_targetSdkVersion=31
...
 
 # workaround for now, used for HMS
-Airship_airshipVersion=16.11.1
+Airship_airshipVersion=7.0.1
```
Comment (rlepinski) [inline android/gradle.properties:10]: bump to 7.0.2

File: react-native-airship.podspec [hunk 3]
```diff
@@ -36,6 +36,6 @@ Pod::Spec.new do |s|
   
   
 
-  s.dependency "AirshipFrameworkProxy", "2.1.1"
+  s.dependency "AirshipFrameworkProxy", "3.0.0"
```
Comment (rlepinski) [inline react-native-airship.podspec:39]: Bump to 3.0.1

**Review by jyaganeh — APPROVED** [review]
:+1: pending version bumps

**Pull request merged**